android/versioned-abis/expoview-abi49_0_0/src/main/java/abi49_0_0/host/exp/exponent/core/modules/ExpoGoUpdatesModule.kt

```diff
@@ -43,7 +43,7 @@ class ExpoGoUpdatesModule(experienceProperties: Map<String, Any?>) : Module() {
         constants["runtimeVersion"] = configuration.runtimeVersionRaw ?: ""
         constants["checkAutomatically"] = configuration.checkOnLaunch.toJSString()
         constants["channel"] = configuration.requestHeaders["expo-channel-name"] ?: ""
-        constants["nativeDebug"] = BuildConfig.EX_UPDATES_NATIVE_DEBUG
+        constants["nativeDebug"] = BuildConfig.DEBUG
```

Comment:
Nice catch!

```suggestion
        constants["nativeDebug"] = false
```
(for expo go having this field doesn't even make sense for the most part, I'm fine just hardcoding)